lib/diaspora_federation/entities/relayable.rb

```diff
@@ -28,6 +28,10 @@ def self.included(model)
           #   a target pod.
           #   @return [String] author signature
           property :author_signature, default: nil
+
+          def self.verify_signatures(data)
```

Comment:
I changed [this](https://github.com/SuperTux88/diaspora_federation/pull/18/files#diff-de8e01cf36c204663baaab85af6af826R102) line to make it support every type of objects that respond to ```verify_signatures ```. Maybe changing Relayable.verify_signatures to class methods of relayable children would be good then to make it respective.